automatic/paint.net/tools/ChocolateyInstall.ps1

```diff
@@ -1,20 +1,27 @@
-﻿$ErrorActionPreference = 'Stop'
+$ErrorActionPreference = 'Stop'
 
-$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+$toolsPath = Split-Path -parent $MyInvocation.MyCommand.Definition
 
 $packageArgs = @{
-  PackageName    = $env:ChocolateyPackageName
-  fileType       = 'exe'
-  file64         = Get-Item $toolsPath\*.exe
-  silentArgs     = '/auto'
+  packageName = $env:ChocolateyPackageName
+  file        = gi $toolsPath\*_x32.zip
+  file64      = gi $toolsPath\*_x64.zip
```

Comment:
Please don't do this. Put the filename in the hashtable and avoid using Get-Item. You can do that from the `update.ps1` file.